src/GraphQL/Execution/ExecutionOptions.cs

```diff
@@ -15,7 +15,11 @@ namespace GraphQL
     /// <summary>Configuration options to be passed to <see cref="IDocumentExecuter"/> to execute a query</summary>
     public class ExecutionOptions : IProvideUserContext
     {
-        /// <summary>Schema of graph to use; required<br/><br/>Schema will be initialized if has not yet been initialized.</summary>
+        /// <summary>
+        /// Schema of graph to use; required
```

Comment:
```suggestion
        /// Schema of graph to use; required<br/>
```
There must be two `<br/>` between sentences to create a blank line between paragraphs.